radix.go

```diff
@@ -275,18 +275,19 @@ DELETE:
 	n.leaf = nil
 	t.size--
 
+	// Check if we should delete this node from the parent
+	if parent != nil && len(n.edges) == 0 {
+		parent.delEdge(label)
```

Comment:
Don't we need to fast quit after this? I worry we can do a `delEdge` and then do `mergeChild` at 290 under some conditions